src/Exception/CouldNotInstallModule.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+namespace Mollie\Exception;
+
+class CouldNotInstallModule extends MollieException
+{
+    const FAILED_TO_INSTALL_ORDER_STATE = 1;
```

Comment:
Maybe add ExceptionCode?

Reply:
Ok, in the future exceptions will be refactored so hopefully I will be able to separate main logic and infrastructure.